### PR TITLE
Aravind Annendum Results

### PR DESCRIPTION
**Pull request by aravind-segu**
- Added Imposter Syndrome Section to Health
- Added Interviews Section to Coop

### CODE REVIEW

File: content/health.html [hunk 1]
```diff
@@ -59,6 +59,28 @@ <h3> Mental Health </h3>
 
 <div class="vbx4"></div>
 
+<h3>Imposter Syndrome</h3>
+<p class="half">
+  <span class="big">Over 80% of the respondants felt imposter syndrome over their university years</span>  
+  Impostor syndrome refers to an individual's internal experience of believing that they are not as competent as others perceive them to be. 
+  The Software Engineering Program is very competitive and has one of the highest admission averages in Waterloo. This can place students in a 
+  pressure to achieve environment which might have lead to students feeling like an imposter.
```
Comment (nsabahi) [inline content/health.html]:
```suggestion
  pressure-to-achieve environment which might have lead to students feeling like an imposter.
```

**Review by nsabahi — APPROVED** [review]
LGTM! Small nit, mainly trying to clarify the wording of that sentence

File: content/health.html [hunk 2]
```diff
@@ -59,6 +59,28 @@ <h3> Mental Health </h3>
 
 <div class="vbx4"></div>
 
+<h3>Imposter Syndrome</h3>
+<p class="half">
+  <span class="big">Over 80% of the respondants felt imposter syndrome over their university years</span>
```
Comment (jaxonL) [inline content/health.html]:
```suggestion
  <span class="big">Over 80% of respondents felt imposter syndrome over their university years</span>  
```

File: content/coop.html [hunk 3]
```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
   <span class="graph-title"> Admission average compared to monthly salary</span>
   <div id="admission-salary"></div>
 </div>
+
+<div class="vbx2"></div>
+<h3> Interviews </h3>
+<p class="full">
+  Over the coop interview cycles students sometimes missed an interview or were late to an interview.
```
Comment (jaxonL) [inline content/coop.html]:
```suggestion
  Over the coop interview cycles, students sometimes missed an interview or were late to an interview. 
```

File: content/coop.html [hunk 4]
```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
...
+<div class="half">
+  <span class="graph-title">Missed an Interview</span>
+  <div id="missed_interview" class="graph-center"></div>
+</div>
+<div class="half second">
+    <span class="graph-title">Late to an Interview</span>
+    <div id="late_interview" class="graph-center"></div>
+</div>
+<div class="vb"></div>
+<div class="half">
```
Comment (jaxonL) [inline content/coop.html:263]: loved the stories :D there's also a `path-section` class that was used in misc -> "beauty in the details" section that has some sort of styling for stories, if you wanted to take a look

File: content/coop.html [hunk 5]
```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
...
+  <span class="big">The Phone Interview Mixup</span>
+  <p>
+    "I waited in the TC lobby and at T+5mins my name hadn't shown up so I double checked and it was actually a phone interview.
```
Comment (jaxonL) [inline content/coop.html:266]:
```suggestion
    "I waited in the TC lobby and at T+5mins my name hadn't shown up so I double-checked and it was actually a phone interview. 
```

File: content/coop.html [hunk 6]
```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
...
+    By some miracle I had my laptop in my bag, so I barged in a random office in TC and pleaded to use the room. 
+    Luckily the office worker let me use the room for 30 minutes. Pulled up a chair and somehow aced the interview. 
+    Ended up getting the offer and taking it."
+  </p>
+  <div class="vb"></div>
+  <span class="big">Alarm Mishaps</span>
+  <p>
+    "I missed my alarm and woke up at 8:20 for an interview at 8:30. I quickly got dressed and ran to TC as fast as I could. My name was
+    already called and the receptionist was trying to get a hold of me through my phone. Finally ran up the stairs, 
+    15 mins late to my 30 min interview and burst in to the room. The interviewer was surprisingly very kind and
```
Comment (jaxonL) [inline content/coop.html]:
```suggestion
    15 mins late to my 30 min interview and burst into the room. The interviewer was surprisingly very kind and 
```

File: content/coop.html (hunk 4 above)
Comment (jaxonL) [reply content/coop.html:263]: the below comments are small nits to reading fluidity

File: content/coop.html [hunk 7]
```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
...
+    "I missed my alarm and woke up at 8:20 for an interview at 8:30. I quickly got dressed and ran to TC as fast as I could. My name was
+    already called and the receptionist was trying to get a hold of me through my phone. Finally ran up the stairs, 
+    15 mins late to my 30 min interview and burst in to the room. The interviewer was surprisingly very kind and 
+    understanding and allowed me to catch my breath. Finally, I somehow aced the interview and got the offer."
+  </p>
+</div>
+<div class="half second">
+  <span class="big">Rainy Day Adventures</span>
+  <p>
+    "They requested a paper copy of my resume, so I had to print one. So went to the interview from V1,
```
Comment (jaxonL) [inline content/coop.html]:
```suggestion
    "They requested a paper copy of my resume, so I had to print one. So I went to the interview from V1, 
```

File: content/coop.html [hunk 8]
```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
...
+    15 mins late to my 30 min interview and burst in to the room. The interviewer was surprisingly very kind and 
+    understanding and allowed me to catch my breath. Finally, I somehow aced the interview and got the offer."
+  </p>
+</div>
+<div class="half second">
+  <span class="big">Rainy Day Adventures</span>
+  <p>
+    "They requested a paper copy of my resume, so I had to print one. So went to the interview from V1, 
+    was even going to be early, got to the environment buildings and realized I forgot to print the resume, 
+    so had to go back. Had issues printing, and then when I got back to my room I couldn't find my keys
```
Comment (jaxonL) [inline content/coop.html]:
```suggestion
    so I had to go back. I had issues printing, and then when I got back to my room I couldn't find my keys 
```

File: content/health.html [hunk 9]
```diff
@@ -59,6 +59,28 @@ <h3> Mental Health </h3>
...
+  pressure to achieve environment which might have lead to students feeling like an imposter.
+</p>
+<div class="half second">
+  <span class="graph-title">Students who felt Imposter Syndrome</span>
+  <div id="imposter-syndrome" class="graph-center"></div>
+</div>
+<p class="half">
+  By the end of their university career, around 46% of students do not feel imposter syndrome, while 30% of the students sometimes do and 24% of the 
+  students still do. This shows a growth in the student's confidence over the years as they overcame their academic hurdles, experienced the real world
+  during coops and found their calling in software engineering.
```
Comment (jaxonL) [inline content/health.html]:
```suggestion
  during coops, and found their calling in software engineering. 
```

i'm personally a sucker for the oxford comma, but i would want to know how the rest of us feel haha

@aravind-segu  @nsabahi @roxelo @AkshayPall @jenniferwills  thoughts?

**Review by jaxonL — COMMENTED** [review]
small suggestions for readability, and an oxford comma quip. looking good otherwise :D

**Pull request merged**